.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -1,5 +1,6 @@
 name: 🐞 Bug Report
 description: Report a bug if something isn't working as expected in WooCommerce Core.
+title: "[Bug]: "
```

Comment:
I just noticed we use uppercase [DOC BUG] for the documentation issue template. Should we maybe change to uppercase to match it?

Reply:
```suggestion
title: "[BUG]: "
```